Contribute/index.html

```diff
@@ -53,7 +55,7 @@ <h2 id="code">Code, Proofs, and Documentation</h2>
 <p>
   Note that larger changes, such as an architecture port, or adding a new
   feature to the kernel, requires a <a href="{% link Contribute/rfc-process.md
-  %}">Request for Comments</a> first, in which you describe the change and get
+  %}">Request for Comments (RFC)</a> first, in which you describe the change and get
```

Comment:
```suggestion
  %}">RFC</a> first, in which you describe the change and get
```
(already introduced above)